addons/common/fnc_players.sqf

```diff
@@ -0,0 +1,24 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_players
+
+Description:
+    Reports all (human) player objects. Does not include headless client entities.
+
```

Comment:
Add a short paragraph after the one-liner description that explains that this is different from`allPlayers`/`BIS_fnc_listPlayers`.